cmd/kk/pkg/loadbalancer/tasks.go

```diff
@@ -310,6 +310,13 @@ func (g *DeleteVIP) Execute(runtime connector.Runtime) error {
 	if !ok {
 		return errors.New("get interface failed")
 	}
+	
+	address := host.GetAddress()
+	internalAddress := host.GetInternalAddress()
+	if address == g.KubeConf.Cluster.ControlPlaneEndpoint.Address || internalAddress == internalAddress {
```

Comment:
I don't know what is `internalAddress == internalAddress`, and I guess maybe it is `internalAddress == address`.